cookbooks/swift/templates/default/home/rclone/rclone.erb

```diff
@@ -0,0 +1,79 @@
+[admin]
+type = swift
+env_auth = false
+user = admin:admin
+key = admin
+auth = <%= @ssl ? "https://#{@hostname}" : "http://#{@hostname}:8080" %>/auth/v1.0
+auth_version = 1
+
+[admin-s3]
+type = s3
+env_auth = true
```

Comment:
Why `env_auth = true` for S3? We're explicitly setting creds in the file, do we really want to allow the environment to override?

Reply:
I stole this from the autogenerate ones given to us via our downstream portal... so no idea :P I'll change it.